lib/src/nodes/config_node_path.cc

```diff
@@ -0,0 +1,17 @@
+#include <internal/nodes/config_node_path.hpp>
+
+using namespace std;
+
+namespace hocon {
+
+    config_node_path::config_node_path(path node_path, token_list tokens) :
+        _path(move(node_path)), _tokens(move(tokens)) { }
+
+    token_list config_node_path::get_tokens() const {
```

Comment:
In config_node_include, we are creating a list of tokens in this method (this inherits from abstract_config_node). As is, we can't return a reference from that. Suggestions for compromise?